### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by mcavo**
### Summary
Add navbar fixed top navbar using flexbox

### Screenshot
image

### Trello Card
https://trello.com/c/rGN6Lj4c/13-dise%C3%B1o-de-la-p%C3%A1gina
https://trello.com/c/hEKQM6hz/15-navbar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>vc-react</title>
+    <link rel="shortcut icon" type="image/x-icon" href="./images/favicon.ico"/>
+    <link rel="stylesheet" href="application.css">
+  </head>
+  <body>
+    <div class="page">
```
Comment (mcallegari10) [inline index.html]: This div is not necessary :smile:

File: scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
+@import url(http://fonts.googleapis.com/css?family=Roboto);
+
+// Colors
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
```
Comment (mcallegari10) [inline scss/application.scss]: This is great but move the variables into a separate file. For example in this case:

```
/scss
| -- /variables
     | -- _colors.scss
```

And there you will store all variables  and them you can import them :smile:

Comment (mcallegari10) [reply scss/application.scss]: Do the same for the sizes

File: scss/application.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+$tech-logo-size: 100px;
+
+// Font sizes
+$font-big: 24px;
+$font-medium: 16px;
+$font-small: 12px;
+
+body {
+  margin: 0px 0px 0px 0px;
+  overflow: hidden;
```
Comment (mcallegari10) [inline scss/application.scss]: The equalizer is supposed to resolve this, you should be fine without this rules :smile_cat:

File: scss/application.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+.page {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.page-navbar {
+  height: $nav-height;
+  flex: 0 0 auto;
```
Comment (mcallegari10) [inline scss/application.scss]: No need for this

File: scss/application.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0px 10px 5px $grey;
+  padding-left: 20px;
+  padding-right: 20px;
```
Comment (mcallegari10) [inline scss/application.scss]: Merge the paddings into: `padding: 0 20px;` or `padding: auto 20px;`

File: scss/application.scss [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.page-navbar {
+  height: $nav-height;
+  flex: 0 0 auto;
+  display: flex;
+  flex-flow: row nowrap;
```
Comment (mcallegari10) [inline scss/application.scss]: I think there's no need for this

File: scss/application.scss [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0px 10px 5px $grey;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.page-content {
+  flex: 1 1 auto;
```
Comment (mcallegari10) [inline scss/application.scss]: You can sum up this one with: `flex: 1`

File: scss/application.scss [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  position: relative;
+  overflow-y: auto;
+}
+
+a {
```
Comment (mcallegari10) [inline scss/application.scss]: Avoid styling HTML tags directly. Always use css classes

File: scss/application.scss (hunk 2 above)
Comment (mcallegari10) [reply scss/application.scss]: BTW, this file should import all the styles and then you import this file in the HTML.

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: scss/application.scss [hunk 9]
```diff
@@ -1,26 +1,10 @@
...
-$font-small: 12px;
+@import "variables/colors";
+@import "variables/fonts";
+@import "variables/sizes";
 
 body {
-  margin: 0px 0px 0px 0px;
-  overflow: hidden;
   font-family: 'Roboto',serif;
+  margin: 0px 0px 0px 0px;
```
Comment (mcallegari10) [inline scss/application.scss]: If you really really really need this, change to: `margin: 0;`

**Review by mcallegari10 — APPROVED** [review]
Check that please :smile:

File: scss/application.scss (hunk 9 above)
Comment (mcavo) [reply scss/application.scss]: I do :sweat_smile:

**Pull request merged**